### ISSUE TITLE
Recovery codes.txt name 

### ISSUE THREAD

**Issue by SorinaFlorean** [mozilla/fxa-content-server#6071]
STR: 
Download 3-4 recovery codes files from different accounts.

Actual results: the files are named Recovery codes(1), Recovery codes(2), Recovery codes(3).
Expected results: name of the account is added to the .txt file. 

See the screenshot: https://i.imgur.com/Ej5sgAi.png.

**Comment by shane-tomlinson** [comment]
I agree, this seems a bit strange. I think "Recovery codes.txt" is a bit generic, perhaps something like "Firefox Recovery Codes <email> <date>.txt"
